pkg/build/api/types.go

```diff
@@ -59,6 +59,10 @@ type BuildSpec struct {
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
 
+	// PostCommit is a build hook executed after the build output image is
+	// committed, before it is pushed to a registry.
+	PostCommit BuildPostCommitSpec
```

Comment:
It is `postCommit` because not long ago we we were about to add `post`, which meant `postPush` and would trigger downstream builds. And longer ago, we were about to add "post build hooks" which were also semantically after push. Since the "push" part of the build can fail, I see value in being able to have post push actions at some point.